### ISSUE TITLE
Complete h-entry for posts

### ISSUE THREAD

**Issue by blineberry** [blineberry/orangegnome#21]
Add author
```
<a rel="author" class="p-author h-card" href="…">Your Name</a>
```

Add url
```
<a class="u-url" href="…">…</a>
```

Add categor(y|ies)
<a class="p-category" href="…">…</a>

**Comment by blineberry** [comment]
https://indiewebify.me/validate-h-entry/?url=https%3A%2F%2Forangegnome.com%2Fposts%2F84%2Fold-wordpress-blog-migrated-to-new-django-site

**Comment by blineberry** [comment]
Fixed by #25
